packages/block-editor/src/hooks/color.js

```diff
@@ -181,12 +182,10 @@ const getLinkColorFromAttributeValue = ( colors, value ) => {
  */
 export function ColorEdit( props ) {
 	const { name: blockName, attributes } = props;
-	const { colors, gradients, __experimentalEnableLinkColor } = useSelect(
-		( select ) => {
-			return select( 'core/block-editor' ).getSettings();
-		},
-		[]
-	);
+	const isLinkColorEnabled = useEditorFeature( 'color.link' );
```

Comment:
I really like how the `useEditorFeature` simplifies the code. Can't wait to see the same done for presets.